### PR TITLE
Add a blog post conveying scope and scale of usage

### PR DESCRIPTION
**Pull request by juliakreger**

### CODE REVIEW

File: src/pages/blog/scale-of-usage.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+<br>
+Which sure seems like a similar conundrum, and establishing the context of who the customer happens to be.
+
+## Artifact of infrastucture ##
+
+The closer you get to lower layers of intrastructure, the more important it becomes. Also, the greater impact one can have should details make it into the eyes of those who would do harm.
+
+Typically in communities, we can see the power lines, we might be able to figure out where the water lines are precisely, but the closer we get to those infrastucture centers, the more secretive or restrictive those environments need to be to ensure the service and security of the services being provided.
+
+It is the same for cloud infrastucture operators, telecommunications providers, and even train operators. They provide a substrate service to meet the needs of their customers. Often, the customer never knows how the server got there. Or how their telephone call completed, or what drove the spike into the railroad track tie.
```
Comment (arnewiebalck) [inline src/pages/blog/scale-of-usage.md:32]: Nit: infrastructure

File: src/pages/blog/scale-of-usage.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+But, we *can* piece together public context, and this is why the [OpenStack Baremetal Logo program](https://www.openstack.org/use-cases/bare-metal/) is helpful to the Ironic community. It helps highlight the firms which have an interest in the software. To join the program, each organization was required to submit their use case in how they use ironic. A number of them resulted in blog posts and stories. While this doesn't always result in explicit counts of number of servers, it is only the portion who actively stepped forward.
+
+Another avenue for guaging interest and usage is the actual upstream project is through who contributes. We don't necessarilly have context into thier usage or use case unless they speak of it publically, but a commit from a firm may be be driven by any number of different drivers: Customer needs, Internal Need, Bugfixes from internal or external usage, Compelled need to fix a typo, or a friend just wanting to make sure they can have breakfast on occassion with another friend resulting in a pain point being fixed.
```
Comment (arnewiebalck) [inline src/pages/blog/scale-of-usage.md:48]: Nit: gauging

Comment (arnewiebalck) [inline src/pages/blog/scale-of-usage.md:48]: Nit: is who contributes to the actual upstream project.

Comment (arnewiebalck) [inline src/pages/blog/scale-of-usage.md:48]: Nit: necessarily

Comment (arnewiebalck) [inline src/pages/blog/scale-of-usage.md:48]: Nit: publicly

**Review by arnewiebalck — CHANGES_REQUESTED** [review]
Thanks, Julia, this is great! I found some typos.

File: src/pages/blog/scale-of-usage.md (hunk 1 above)
Comment (juliakreger) [reply src/pages/blog/scale-of-usage.md:32]: Wow, multiple times :(

File: src/pages/blog/scale-of-usage.md (hunk 2 above)
Comment (juliakreger) [reply src/pages/blog/scale-of-usage.md:48]: Good point, done.

Comment (juliakreger) [reply src/pages/blog/scale-of-usage.md:48]: Done.

Comment (juliakreger) [reply src/pages/blog/scale-of-usage.md:48]: done

**Review by arnewiebalck — APPROVED** [review]

**Pull request merged**